src/Conference/ConferenceDetails.jsx

```diff
@@ -44,6 +44,14 @@ const ConferenceDetails = ({ conference, id, isLoading, error }) => {
     deleteConference(id);
   };
 
+  const expressInterest = async () => {
+    try {
+      await favouriteConference(conference, token);
+    } catch (error) {
+      console.log(error);
```

Comment:
Is there much use for this log? Might be nice to use the toast here for displaying the error

Reply:
Ok! I have a toast for this in the axios request, so should I just remove the console log and leave the catch part blank?

Reply:
You can just remove the whole try catch and just write the `await favouriteConference(conference, token);` line I believe - let me know if it's working funny and I'll come over and help.